src/index.js

```diff
@@ -1,8 +1,21 @@
 import { applyMiddleware } from 'redux'
-import isPlainObject from 'lodash.isplainobject'
 
 const isFunction = arg => typeof arg === 'function'
 
+const isPlainObject = function (obj) {
+  if (!isObjectLike(obj) || getTag(obj) != '[object Object]') {
```

Comment:
where is `isObjectLike` defined ?
It will throw an error

Reply:
same for `getTag`

Reply:
Sorry I copied the wrong code.